gnomad_constraint/constraint/constraint_pipeline.py

```diff
@@ -0,0 +1,63 @@
+# noqa: D100
+
+import argparse
+import logging
+
+import hail as hl
+
+from gnomad.resources.grch38.reference_data import vep_context
+from gnomad.resources.grch37.gnomad import public_release
+from gnomad_constraint.utils.constraint_basics import (
+    add_vep_context_annotations,
+    prepare_ht_for_constraint_calculations,
+)
+
+from gnomad_constraint.resources.resource_utils import get_processed_ht_path
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("constraint_pipeline")
+logger.setLevel(logging.INFO)
+
+
+def main(args):
+    """Execute the constraint pipeline."""
+    trimers = args.trimers
+    if args.pre_process_data:
+        add_vep_context_annotations(
```

Comment:
it's help to have a logger to print out text describing when step is currently being run